packages/google_maps_flutter/google_maps_flutter_android/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.1.11
```

Comment:
Only a couple of packages should have had a version change (not including this one). It looks like maybe your base-sha was determined incorrectly; it's supposed to Just Work, but some people have been having issues.

If you run
```
git co 00f464f4e1673698de939762a8a88cd138130ee3 -- packages/*/*/CHANGELOG.md packages/*/*/pubspec.yaml
git commit -m "revert metada changes"
dart script/tool/bin/flutter_plugin_tools.dart update-release-info --version=minimal --base-sha=00f464f4e1673698de939762a8a88cd138130ee3 --changelog='Fixes violations of new analysis option use_named_constants.'
```

it should come out right. I'll update the wiki to discuss `base-sha` since this has come up a few times now.

Reply:
I thought this seemed wrong, I'll get it fixed